taxi_service/settings.py

```diff
@@ -53,10 +54,12 @@
 
 ROOT_URLCONF = "taxi_service.urls"
 
+BASE_DIR = Path(__file__).resolve().parent.parent
```

Comment:
Redundancy issue: You have defined the BASE_DIR variable twice in this file (lines 17 and 57). You only need to define it once at the beginning of your settings file.